src/classes/Cart.ts

```diff
@@ -231,7 +231,11 @@ abstract class Cart {
 
         this.offer.data('handleTimestamp', moment().valueOf());
 
-        this.offer.setMessage('Powered by TF2 Automatic');
+        if (process.env.OFFER_MESSAGE == undefined || process.env.OFFER_MESSAGE == ' ') {
```

Comment:
You can change this to `!process.env.OFFER_MESSAGE` to check for both cases, after changing the default value to an empty string.

_Note: in the future you should use `===` instead of `==` in most cases for better performance_

Reply:
I've tried to put "!" in front of `process.env.OFFER_MESSAGE` but when compiling, it failed.
I'll change the `==` to `===`.
![compile failed](https://user-images.githubusercontent.com/47635037/75605039-ffac1880-5b19-11ea-9d1f-5dd83a8086cb.PNG)

Reply:
Maybe typescript doesn't allow that then, I haven't really used it yet

Reply:
also changing "==" to "===" seems problematic with CLI checks. 🤔